doctr/models/detection/differentiable_binarization/base.py

```diff
@@ -90,6 +90,8 @@ def bitmap_to_boxes(
         Args:
             pred: Pred map from differentiable binarization output
             bitmap: Bitmap map computed from pred (binarized)
+            angle_tol: Tolerancy to compare the angle to the median angle of the page
```

Comment:
typo "Tolerancy to compare..." --> "Comparison tolerance of the angle with the median angle across the page"